src/03_modules.py

```diff
@@ -7,10 +7,9 @@
 
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
-
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(sys.argv[0])
```

Comment:
I don't think you finished this section.

Also - this problem is wanting you to print out each argument from the command line on a new line. So if you had python3 03_modules.py dog cat bird, it would print
03_modules.py
dog
cat
bird

I would recommend using `for arg is sys.argv: print(arg)`